pandas/tests/test_categorical.py

```diff
@@ -3943,6 +3943,38 @@ def f():
                                   'category', categories=list('cab'))})
         tm.assert_frame_equal(result, expected)
 
+    def test_union(self):
```

Comment:
does that belong to the concat tests or the categoricals? The API is in `concat` and not in `Categorical` (which I like... :-) )